src/components/Widgets/DateControl.js

```diff
@@ -0,0 +1,21 @@
+import React, { PropTypes } from 'react';
+import DateTime from 'react-datetime';
+
+export default class DateControl extends React.Component {
+  handleChange = (datetime) => {
+    this.props.onChange(datetime);
+  };
+
+  render() {
+    return (<DateTime
+      timeFormat={false}
+      value={this.props.value || new Date()}
+      onChange={this.handleChange}
+    />);
+  }
+}
+
+DateControl.propTypes = {
+  onChange: PropTypes.func.isRequired,
+  value: PropTypes.object,
```

Comment:
The reason we currently have a tighter coupling with moment is because in general it's so important to have timezone, date formating functions, etc, throughout the CMS that the standard Date model is not enough, and that the YAML formatter uses it as part of the serialization/deserializtion process right now as the only way to know how a date should be formattet in the YAML (for Jekyll for example, it makes a huge different if a date is persisted like `2016-11-01` or `2016-11-01 00:00:00` since once will get deserialized as a Ruby Date object, the other as a Ruby DateTime object and those behave differently).

The Ember prototype uses moment.js for deserialized dates since moment can carry around the format in the object - but I think we've actually broken that functionality in the React version right now...